security/fma-operator-fee.md

```diff
@@ -0,0 +1,115 @@
+- **Risk Assessment:**
+  High impact, low likelihood.
+  **Mitigations:**
+  Before setting or updating the operator fee params, the operator should carefully read the [corresponding specs](https://specs.optimism.io/protocol/isthmus/exec-engine.html#operator-fee) and simulate the impact of operator fee on the whole transaction cost.
+- **Detection:** 
+  Monitor the transaction costs and alert if it's above a threshold.
+- **Recovery Path(s)**:
+  If the operator fee parameters are set to unreasonable values, the rollup operator should update the `operatorFeeScalar` and `operatorFeeConstant` to reasonable values as soon as possible.
+
+### FM2: Broken Fee Estimation (Wallets)
+
+- **Description:** 
+  If wallets fail to update their fee estimation logic, users will no longer be shown the accurate costs of a transaction.
+- **Risk Assessment:**
+  Medium impact, medium likelihood.
+  **Mitigations:**
+  Coordinate with wallet providers to update their fee estimation logic. This includes MetaMask, Coinbase Wallet, and others.
+- **Detection:** 
+  Confirm that wallets are using the correct fee estimation logic post-launch. This can be done manually on chains that have added an operator fee.
+- **Recovery Path(s)**:
+  Notify wallets of the new fee structure and ask them to update their fee estimation logic if the operator fee is enabled.
+
+### FM3: Bug in Receipt Hydrating Logic
+
+- **Description:**
+  If there is a bug in the receipt hydrating logic, the operator fee may not be correctly reflected in transaction receipts, leading to incorrect fee reporting and potential accounting issues.
+- **Risk Assessment:**
+  Medium impact, low likelihood.
+- **Mitigations:**
+  Action and E2E tests covering the receipt hydration logic has been added.
+
+  * [Fee E2E test](https://github.com/ethereum-optimism/optimism/blob/develop/op-e2e/system/fees/fees_test.go)
+  * [Operator Fee Constistency action test](https://github.com/ethereum-optimism/optimism/blob/develop/op-e2e/actions/proofs/operator_fee_test.go)
+
+- **Detection:**
+  The action or E2E tests or local testing may pick up an issue.
+- **Recovery Path(s):**
+  Deploy fix for receipt hydration logic. Historical receipts will remain incorrect but can be recalculated using on-chain data if needed.
+
+### FM4: Database Growth Impact on Nodes
+
+- **Description:**
+  The addition of operator fee fields increases the size of transaction receipts, leading to faster database growth. This could accelerate the need for solutions like EIP-4444 or other history expiry mechanisms.
+- **Risk Assessment:**
+  Medium impact, high likelihood.
+- **Mitigations:**
+  - Implement history expiry solutions like EIP-4444 when available.
+- **Detection:**
+  - Monitor database growth rate compared to pre operator fee baseline.
```

Comment:
A few of the detections are monitoring related. If we don't have monitors setup for these types of metrics already we will have to stand those up, which would be an action item to track

Reply:
I'll ask if OP Labs habe monitoring in place related to database growth rate.

Reply:
Before we merge, has this been resolved @leruaa?